javalib/src/main/scala/scala/scalanative/nio/fs/windows/WindowsPathParser.scala

```diff
@@ -120,8 +120,8 @@ object WindowsPathParser {
       to: Int,
       allowedChars: String = ""
   ): String = {
-    def isSpaceOrReservedChar(c: Char) = {
-      c.isSpaceChar || pathReservedChars.contains(c)
+    def isControlOrReservedChar(c: Char) = {
+      c < '\u0020' || pathReservedChars.contains(c)
```

Comment:
I would consider either `val space = '\u0020' or leaving a comment at this location to make this a touch more readable. Other than that this looks great.

Reply:
Sure, good idea